components/sliders/SliderProjects/index.tsx

```diff
@@ -0,0 +1,127 @@
+'use client';
+import React, { useState } from 'react';
+import Slider from 'react-slick';
+
+import ArrowLeft from '@/components/sliders/ArrowLeft';
+import ArrowRight from '@/components/sliders/ArrowRight';
+import { DepartmentProject } from '@/types/departments';
+
+import DepartmentCardPicture from './DepartmentCardPicture';
+import DepartmentCardText from './DepartmentCardText';
+import DepartmentMoblieCard from './DepartmentMobileCard';
+import IconBottomLeftCornerCur from './IconBottomLeftCornerCut';
+import SkeletonDepartmentCard from './SkeletonDepartmentCard';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import '@/components/sliders/styles.css';
+
+const SliderProjects = ({
+  projects,
+  loading,
+}: {
+  projects: DepartmentProject[] | undefined;
+  loading: boolean;
+}) => {
+  const [nav1, setNav1] = useState<Slider | null>(null);
+  const [nav2, setNav2] = useState<Slider | null>(null);
+
+  const next = () => {
+    if (!nav1 || !nav2) return;
+    nav1.slickNext();
+    nav2.slickNext();
+  };
+
+  const previous = () => {
+    if (!nav1 || !nav2) return;
+    nav1.slickPrev();
+    nav2.slickPrev();
+  };
+
+  const generalSettingsSlider = {
+    dots: false,
+    infinite: true,
+    speed: 300,
+    waitForAnimate: true,
+    pauseOnHover: true,
+    arrows: false,
+    swipeToSlide: false,
+    slidesToScroll: 1,
+  };
+
+  const settingsForSlider1 = {
+    autoplaySpeed: 7000,
+    autoplay: true,
+    slidesToShow: 1,
+    fade: true,
+  };
+
+  const settingsForSlider2 = {
+    className: 'main-slider',
+    slidesToShow: 3,
+    responsive: [
+      {
+        breakpoint: 1536,
+        settings: {
+          slidesToShow: 1,
+          fade: true,
+        },
+      },
+    ],
+  };
+  if (!projects || loading) {
```

Comment:
`projects` will always be true, check it with `projects.length` and remove loading state